.github/workflows/pull_request.yaml

```diff
@@ -4,44 +4,40 @@ on:
   push:
     branches:
     - 'master'
+    - 'main'
```

Comment:
why both master and main? can master be removed?

Reply:
File copied directly from main branch

Reply:
eh removed